Community-Committee-Charter.md

```diff
@@ -0,0 +1,128 @@
+
+## Section 4. Establishment of the Community Committee.
+
+Membership is for 6 months. The group will ask on a regular basis if the expiring members would like to stay on. A member just needs to reply to renew. There is no fixed size of the CC. However, the expected target for Advisors, as defined in Section 10, is between 9 and 12, to ensure adequate coverage of important areas of community expertise, balanced with the ability to make decisions efficiently.
+
+There is no specific set of requirements or qualifications for CC membership beyond these rules. The CC may add additional members to the CC by a standard CC motion and vote. A CC member may be removed from the CC by voluntary resignation, or by a standard CC motion.
+
+Changes to CC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+No more than one-fourth of the CC members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CC member, or a change of employment by a CC member, creates a situation where more than one-fourth of the CC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CC members affiliated with the over-represented employer(s).
+
+The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
+
+The CC may, at its discretion, invite any number of non-voting Guests to participate in the public portion of CC discussions and meetings.
+
+The CC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CC). The meeting shall be directed by the Individual Membership Directors. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+CC members are expected to regularly participate in CC activities.
+
+In the case where an individual CC member -- within any three month period -- attends
+fewer than 25% of the regularly scheduled meetings, does not participate in CC
+discussions, and does not participate in CC votes, the member shall be automatically
+removed from the CC. The member may be invited to continue attending CC meetings
+as a Guest.
+
+## Section 5. Responsibilities of the CC.
+
+Subject to such policies as may be set by the Board, the CC is responsible for all cultural development and outreach within the Node.js Foundation, including:
+
+- Outreach to community organizations
+- Documentation of community organizations
+- Cultural direction of Node.js Foundation
+- Overseeing the Inclusivity WG
+- Overseeing education initiatives for the Node.js Foundation
+- Project governance and process (including this policy)
```

Comment:
This might imply governance across all of the project.  Should it be "Project governance and process for the areas under the scope of the Community Committe"

Reply:
This is already in the charter approved by the board. Please refer to https://gist.github.com/MylesBorins/356c3f6ecc49bf8aa0b27476400d7805 to see the actual diff